### ISSUE TITLE
compiling error

### ISSUE THREAD

**Issue by kategorically** [hadley/ggplot2-book#85]
I'm using
` R version 3.2.2 (2015-08-14)
Platform: x86_64-apple-darwin13.4.0 (64-bit)
Running under: OS X 10.10.5 (Yosemite)

locale:
[1] en_GB.UTF-8/en_GB.UTF-8/en_GB.UTF-8/C/en_GB.UTF-8/en_GB.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
 [1] httr_1.0.0      R6_2.1.1        magrittr_1.5    rversions_1.0.2
 [5] tools_3.2.2     curl_0.9.3      Rcpp_0.12.0     memoise_0.2.1  
 [9] xml2_0.1.2      stringi_0.5-5   knitr_1.11      git2r_0.11.0   
[13] stringr_1.0.0   digest_0.6.8    devtools_1.8.0`

I've tried compiling the book but kept getting an error:

`Rscript book/render-tex.R mastery.rmd
Quitting from lines 198-206 (mastery.rmd) 
Error: Unknown parameters: NA
Execution halted
make: *** [book/tex/mastery.tex] Error 1

Exited with status 2.1 `

**Comment by mli2830** [comment]
I saw this too, on `R version 3.2.2 (2015-08-14); Platform: x86_64-pc-linux-gnu (64-bit); Running under: Ubuntu 14.04.3 LTS`.  

* First I had trouble installing `Hmisc` when the development version of `ggplot2`, `scales` were already installed (`Hmisc` depends on these packages but doesn't (?) appear to install properly with the development versions).  Backed out to CRAN versions, installed `Hmisc`, re-installed dev versions.
* Then I got the error above. Somehow, magically, the error went away after cycling through (i) installing dev version of `ggplot2` (ii) backing out to CRAN version of `ggplot2` (and getting a different error) (iii) re-installing dev version.  ????  The problem *appears* to be with the definition of `qplot()`, specifically with calls like `qplot(x,y,1:10,...)` -- the `1:10` doesn't appear to be matched by anything sensible?
* After that, I instead got as far as:
```
Rscript book/render-tex.R scales.rmd
Quitting from lines 389-395 (scales.rmd) 
Error: Unknown parameters: size
Execution halted
```
and (after flailing around a little bit more) that's where I'm currently stuck.
```
